### PR TITLE
add manpage for dasm

### PR DESCRIPTION
**Pull request by jnahmias**
View the manpage using the command `man -l docs/dasm.1`.

### CODE REVIEW

**Comment by dionoid** [comment]
DASM currently already shows a list of available arguments when you run `dasm` or `dasm.exe` at the command line.
This works for all platforms (Windows, Linux, macOS), so I don't see the added value of a separate man page.
Also note that DASM is maintained by a group of people, so individual copyright statements in source files is not how we roll :-)

**Comment by jnahmias** [comment]
Hello, thanks for your response! The reason I wrote the manpage is that I am in the process of packaging dasm for the [Debian GNU/Linux distribution](https://debian.org/), and a manpage is required by [Debian Policy](https://www.debian.org/doc/debian-policy/ch-docs.html#manual-pages) for each executable. I wanted to offer the manpage back upstream so that others can benefit from it. I'm happy to license it in any way that works for the project as a whole.
If you're not interested, that's fine; I will just maintain it within the Debian package.

**Comment by andrew-davie** [comment]
I'm thinking that a debian package is a good thing, and I understand the requirement for a manpage
The only issue I have is the inclusion of a copyright notice to yourself;  these days we consider DASM a group effort - even though individuals may indeed work on parts of it.  For example, I spent weeks and weeks working on the 80+ page manual but the copyright is to the group, not myself as an individual. I guess I'm sensitive to copyright notices because recently I spotted a dasm fork which *backdated* copyright ownership to the fork "owner" to 10 years ago. (!!)
In a nutshell, I think the general consensus is that we no longer do individual copyrights on dasm code/documentation/features - it just becomes a mess of complexity in what is, essentially, a totally open-source and free project..

**Comment by dionoid** [comment]
Hi @jnahmias, thanks for adding some context to your pull request. If a man page for DASM has to be maintained somewhere, it indeed makes sense to maintain it in the DASM repo itself.

Could you remove the copyright notice to yourself and replace the ".Sh AUTHOR" by these lines:
`.Sh AUTHORS`
`Written and maintained by the DASM team and its contributors`

**Comment by jnahmias** [comment]
@dionoid Works for me. Updated.
